titus-server-master/src/main/java/com/netflix/titus/master/mesos/DefaultV3TaskInfoFactory.java

```diff
@@ -140,14 +140,14 @@ public DefaultV3TaskInfoFactory(MasterConfiguration masterConfiguration,
         }
 
         // Configure agent job attributes
-        containerInfoBuilder.setAllowCpuBursting(Boolean.parseBoolean(attributes.get(JOB_ATTRIBUTES_ALLOW_CPU_BURSTING)));
-        containerInfoBuilder.setAllowNetworkBursting(Boolean.parseBoolean(attributes.get(JOB_ATTRIBUTES_ALLOW_NETWORK_BURSTING)));
-        containerInfoBuilder.setBatch(Boolean.parseBoolean(attributes.get(JOB_ATTRIBUTES_BATCH)));
+        containerInfoBuilder.setAllowCpuBursting(Boolean.parseBoolean(containerAttributes.get(JOB_PARAMETER_ATTRIBUTES_ALLOW_CPU_BURSTING)));
```

Comment:
should we separate job level attributes from container level? I have a slight preference for different namespaces here, but it's not a strong opinions: `parameter.titus.container.*`.

Reply:
Today, they are namespaced by the `agent` so I would think it would be `parameter.titus.agent` in your example